runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/counters/CounterUpdateAggregators.java

```diff
@@ -18,11 +18,11 @@
 package org.apache.beam.runners.dataflow.worker.counters;
 
 import com.google.api.services.dataflow.model.CounterUpdate;
-import com.google.common.collect.ImmutableMap;
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import org.apache.beam.runners.dataflow.worker.MetricsToCounterUpdateConverter.Kind;
+import org.apache.beam.vendor.guava.v26_0_jre.com.google.common.collect.ImmutableMap;
```

Comment:
Is this checkstyle check disabled for the Dataflow worker?  https://github.com/apache/beam/blob/master/sdks/java/build-tools/src/main/resources/beam/checkstyle.xml#L102

Reply:
Yes, I see it's disabled.
https://github.com/apache/beam/blob/c2f0d28/runners/google-cloud-dataflow-java/worker/build.gradle#L111-L113